composables/useProduct.ts

```diff
@@ -26,6 +26,20 @@ export const useProduct = (referenceKey: string) => {
     }
     loading.value = false
   }
+  const getProductLocationInventory = async (productCode: string, locationCodes: string) => {
```

Comment:
this should be its own composable

Reply:
Composable for ProductLocationInventory is created.